### ISSUE TITLE
Extend the GCalendar node

### ISSUE THREAD

**Issue by danielepolencic** [uasabi/noderosso#40]
_TL;DR: extend the GCalendar node to include listing gmail messages._

## Messages

```ts
{
  topic: 'LIST_EMAILS.V1',
  payload: {
    after?: string // should be parsed with chrono-node
    before?: string // should be parsed with chrono node
    query: string
  }
}
```

If after and before are used, the values should be converted in seconds. Notice the big red warning on the Google API about the timezone: https://developers.google.com/gmail/api/guides/filtering

## Events

Ideally, the same event as this: https://github.com/uasabi/noderosso/blob/a266f2e4c4ddec4757bc3ab6a530b70b12327ba5/nodes/imap-reader/imap-reader.common.ts#L21-L29

## Notes

- Create a copy of the node with the changes above and call it `GoogleAPIs`. We will deprecate GCalendar in favour of this new node.
- Please get in touch and I will share a new set of credentials (unless you want to create your own project in GCP).
